checkpointctl.go

```diff
@@ -293,5 +311,9 @@ func memparse(cmd *cobra.Command, args []string) error {
 	}
 	defer cleanupTasks(tasks)
 
+	if pID != 0 {
```

Comment:
Should we extract only the relevant `pagemap-{PID}.img`, `pages-{PID}` and `mm-{PID}.img` files from the checkpoint archive when `--pid` has been specified? In addition, we could show appropriate error message if the specified PID does not exist in the checkpoint.

Reply:
Yes, I was thinking about this also. To do this, I need to make a change in `crit` (add a getter function to get the value of `pageNo` ).

Reply:
> To do this, I need to make a change in crit (add a getter function to get the value of pageNo). Should i open a PR on go-criu now?

It might be worth implementing this because it can significantly improve the performance for large container checkpoints. Also, when the `--pid` option is not used we don't need to extract `pages-` and `mm-`.